geode-core/src/main/java/org/apache/geode/internal/net/SocketCreator.java

```diff
@@ -990,6 +993,7 @@ public void configureServerSSLSocket(Socket socket) throws IOException {
               ex);
           throw ex;
         }
+        // else ignore
```

Comment:
Does this refer to a "live" if-statement?

Reply:
I'm not sure what you mean by "live". It refers to the if before it. I'm not sure that's what we really should want to do, but it's the current behaviour.